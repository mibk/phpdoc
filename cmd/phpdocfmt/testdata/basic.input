<?php

   /**
      @template T    of Foo \ IBar preferably
  @method     int getBaz(  string ... $names) gets baz*/


thin class () {
	const A = <<<END
/**   @do  NOT  TOUCH
END;
	const B = '   no \\n touching */';

  /** @param   T  $x*/
	public function setBaz(mixed $x): void {
	}
}
