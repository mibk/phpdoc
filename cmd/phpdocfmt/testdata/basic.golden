/**
 * @template T of Foo\IBar                preferably
 * @method   int getBaz(string ...$names) gets baz
 */
thin class () {
	/** @param T $x */
	public function setBaz(mixed $x): void {
	}
}
